doc/api/esm.md

```diff
@@ -361,8 +361,8 @@ from which to resolve from:
 import.meta.resolve('./dep', import.meta.url);
 ```
 
-This function is asynchronous because the ES module resolver in Node.js is
-allowed to be asynchronous.
+This function is synchronous because the ES module resolver in Node.js is
+synchronous.
```

Comment:
. . . and also because `import.meta.resolve` is synchronous in browsers and we want to allow code that uses `import.meta.resolve` to work in Node, using custom `resolve` hooks that users may have registered.

Reply:
Are you suggesting adding something to the effect that it aligns with browser behaviour?

Reply:
Yeah. That’s our motivation for this change, after all. And `import.meta.resolve` _looks_ like a language feature; it would be confusing to users if it behaved differently in Node as in browsers (see `setTimeout`).

Reply:
Are we changing `--experimental-import-meta-resolve` to sync as part of this same PR? Perhaps we should?

Reply:
> Are we changing `--experimental-import-meta-resolve` to sync as part of this same PR? Perhaps we should?

Yes, it does already (see notable changes list in the PR description 🙂)

Reply:
> Yes, it does already (see notable changes list in the PR description 🙂) or am I misunderstanding what you mean?

No, I just didn’t read 😄 Thanks!

Reply:
done: 539f7a5